addons/wowl/static/src/components/dropdown/dropdown.ts

```diff
@@ -0,0 +1,159 @@
+import { Component, useState } from "@odoo/owl";
+import { OwlEvent } from "@odoo/owl/dist/types/core/owl_event";
+import { DropdownItemClickedPayload, ParentClosingMode } from "./dropdown_item";
+
+export class Dropdown extends Component {
+  static template = "wowl.Dropdown";
+  static props = {
+    open: {
+      type: Boolean,
+      optional: true,
+    },
+    hover: {
+      type: Boolean,
+      optional: true,
+    },
+    hoverOpenDelay: {
+      type: Number,
+      optional: true,
+    },
+    hoverCloseDelay: {
+      type: Number,
```

Comment:
It think it is number without initial cap

Reply:
'number' only refers to a type, but is being used as a value here.ts(2693)